user-guide/aragon-court/aragon-court-faq.html

```diff
@@ -177,37 +177,103 @@ <h4 id="anj"><a href="#anj">ANJ</a></h4>
 	<a href="https://wiki.aragon.org/network/aragon_network_juror/" target="_blank">Aragon Network Juror page</a> of the Aragon Wiki.</li>
 </ul>
 <h4 id="governance"><a href="#governance">Governance</a></h4>
 <ul>
 	<li>Which parameters of the Court can be changed? How?
 	<p>
-		 Almost all parameters, except for the term duration, period duration, and Court start time can be changed. In particular:
+		 Almost all parameters, except for the term duration, period duration, and Court start time can be changed without having to re-deploy the court contracts. In particular:
 	</p>
 	<ul>
-		<li>Juror fees</li>
-		<li>Draft fees</li>
-		<li>Settlement fees</li>
-		<li>Argument submission period</li>
-		<li>Commit period</li>
-		<li>Reveal period</li>
-		<li>Appeal period</li>
-		<li>Appeal confirm period</li>
-		<li>Penalty percentage of min active balance to be slashed for losing jurors</li>
-		<li>Final round fees reduction percentage</li>
-		<li>First round jurors number</li>
-		<li>Factor used to increase jurors number in each appeal</li>
-		<li>Number of regular rounds (before final round with all jurors is triggered)</li>
-		<li>Number of terms that a coherent juror in a final round is disallowed to withdraw (to discourage 51% attacks)</li>
-		<li>Multiple of dispute fees required to appeal a preliminary ruling</li>
-		<li>Multiple of dispute fees required to confirm appeal</li>
-		<li>Minimum amount of juror tokens that can be activated</li>
+		<li>Controller contract:<br>
+		<ul>
+			<li>Court parameters:<br>
+			<ul>
+				<li>Juror fees</li>
+				<li>Draft fees</li>
+				<li>Settlement fees</li>
+				<li>Argument submission period</li>
+				<li>Commit period</li>
+				<li>Reveal period</li>
+				<li>Appeal period</li>
+				<li>Appeal confirm period</li>
+				<li>Penalty percentage of min active balance to be slashed for losing jurors</li>
+				<li>Final round fees reduction percentage</li>
+				<li>First round jurors number</li>
+				<li>Factor used to increase jurors number in each appeal</li>
+				<li>Number of regular rounds (before final round with all jurors is triggered)</li>
+				<li>Number of terms that a coherent juror in a final round is disallowed to withdraw (to discourage 51% attacks)</li>
+				<li>Multiple of dispute fees required to appeal a preliminary ruling</li>
+				<li>Multiple of dispute fees required to confirm appeal</li>
+				<li>Minimum amount of juror tokens that can be activated</li>
+			</ul>
+			</li>
+			<li>Governor addresses:
+			<ul>
+				<li>Modules governor</li>
+				<li>Funds governor</li>
+				<li>Configuration governor</li>
+			</ul>
+			</li>
+			<li>Eject governor addresses i.e. remove without replacing:<br>
+			<ul>
+				<li>Funds governor</li>
+				<li>Modules governor</li>
+			</ul>
+			</li>
+			<li>Module addresses:
+			<ul>
+				<li>Dispute Manager</li>
+				<li>Jurors Registry</li>
+				<li>Voting</li>
+				<li>Treasury</li>
+				<li>Subscriptions</li>
```

Comment:
```suggestion
				<li>Subscriptions</li>
				<li>Set new module addresses</li>
```

Reply:
What is `<li>Set new module addresses</li>`?

Reply:
It has the ability not just to change the existing modules' address, but to name new ones

Reply:
New modules? So in addition to the five existing modules, more modules could be added?

Reply:
Exactly, much better that way :)